examples/k8s.yaml

```diff
@@ -96,6 +96,16 @@ provision:
   script: |
     #!/bin/bash
     set -eux -o pipefail
+    # To install a worker node, fill in and uncomment this line:
```

Comment:
Needs more detailed usage
- How to start the control plane node, with vmnet
- How to get the `K8S_URL`, `K8S_TOKEN`, and `K8S_HASH` values

Reply:
I don't think vmnet works with Linux, for instance